src/Microsoft.IdentityModel.Tokens.Saml/GlobalSuppressions.cs

```diff
@@ -28,4 +28,5 @@
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Exception is returned as string", Scope = "member", Target = "~M:Microsoft.IdentityModel.Tokens.Saml.SamlSecurityTokenHandler.ValidateSignature(Microsoft.IdentityModel.Tokens.Saml.SamlSecurityToken,System.String,Microsoft.IdentityModel.Tokens.TokenValidationParameters)~Microsoft.IdentityModel.Tokens.Saml.SamlSecurityToken")]
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Breaking change", Scope = "member", Target = "~P:Microsoft.IdentityModel.Tokens.Saml.SamlAssertion.CanonicalString")]
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Breaking change", Scope = "member", Target = "~P:Microsoft.IdentityModel.Tokens.Saml2.Saml2Assertion.CanonicalString")]
-
+[assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Exception is returned as string", Scope = "member", Target = "~M:Microsoft.IdentityModel.Tokens.Saml.SamlSecurityTokenHandler.ValidateTokenAsync(System.String,Microsoft.IdentityModel.Tokens.TokenValidationParameters)~System.Threading.Tasks.Task{Microsoft.IdentityModel.Tokens.TokenValidationResult}")]
```

Comment:
>Exception is returned as string [](http://example.com/codeflow?start=101&length=31)

consider something along the lines of: "Catching all exceptions to match old model, in new models, we should try to avoid this behavior"